Tasks/NuGetCommand/task.json

```diff
@@ -325,6 +325,28 @@
             "visibleRule": "versioningScheme = byPrereleaseNumber"
         },
         {
+            "name": "packTimezone",
+            "type": "pickList",
+            "label": "Time zone",
+            "defaultValue": "utc",
+            "helpMarkDown": "Specifies the desired time zone used to produce the version of the package.",
+            "required": "false",
+            "groupName": "packOptions",
+            "options": {
+                "utc": "UTC",
+                "local": "Agent local time"
```

Comment:
How will this work on hosted agents? Perhaps clarify whatever timezone the hosted agents use.